superset-frontend/src/views/CRUD/alert/AlertReportModal.tsx

```diff
@@ -516,11 +517,10 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
       }
     });
 
-    const shouldEnableForceScreenshot = contentType === 'chart' && !isReport;
     const data: any = {
       ...currentAlert,
       type: isReport ? 'Report' : 'Alert',
-      force_screenshot: shouldEnableForceScreenshot || forceScreenshot,
+      force_screenshot: !isReport || forceScreenshot, // alerts (!isReport) should always bypass cache
```

Comment:
Wait... for alerts with dashboards we don't want to always bypass the cache.

Reply:
Why not? Is it because of the perf issue that may arise?